docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -138,6 +138,7 @@ dependencies:
 
 *  bcrypt_
 *  docutils_
+*  enum34_ 2.7-3.3
```

Comment:
2.7+ below refers to the version of jinja2, we can change this to "(Python 2 only)"